framework/test/mocha/unit/modules/chain/transport.js

```diff
@@ -801,40 +801,30 @@ describe('transport', () => {
 				describe('when broadcast is defined', () => {
 					beforeEach(async () => {
 						__private.broadcaster = {
-							maxRelays: sinonSandbox.stub().returns(false),
 							enqueue: sinonSandbox.stub(),
 						};
 						transportInstance.onSignature(SAMPLE_SIGNATURE_1, true);
 					});
 
-					it('should call __private.broadcaster.maxRelays with signature', async () => {
-						expect(__private.broadcaster.maxRelays.calledOnce).to.be.true;
+					it('should call __private.broadcaster.enqueue with {} and {api: "postSignatures", data: {signature: signature}} as arguments', async () => {
+						expect(__private.broadcaster.enqueue.calledOnce).to.be.true;
```

Comment:
Please, remove async and return from the test.

Reply:
It seems that I cannot remove them. I get these eslint errors:

```
809:133  error  Unexpected synchronous test  mocha/no-synchronous-tests
866:113  error  Unexpected synchronous test  mocha/no-synchronous-tests
879:110  error  Unexpected synchronous test  mocha/no-synchronous-tests
```